docs/CHANGELOG.rst

```diff
@@ -42,10 +45,27 @@ Changed
 - **BACKWARD INCOMPATIBLE:** Count fragments (or templates) instead of reads by
   default in ``featureCounts`` process and ``BBDuk - STAR - featureCounts``
   pipeline. The change applies only to paired-end data.
+- **BACKWARD INCOMPATIBLE:** Change Docker image in ``alignment-subread``
+  from ``resolwebio/legacy:1.0.0`` with Subread (v1.5.1) to ``resolwebio/rnaseq:3.2.0``
+  with Subread (v1.6.0). ``--unique_reads`` option was removed and ``--multiMapping``
+  option was added. By default aligner report uniquely mapped reads only.
```

Comment:
The change log should convey that the two options are related, e.g. [...] option was added _instead_.